soyiyeon/백준_2446번문제.py

```diff
@@ -0,0 +1,11 @@
+# 예제를 보고 규칙을 유추한 뒤에 별을 찍어 보세요.
+
+# 첫째 줄에 N(1 ≤ N ≤ 100)이 주어진다.
+
+N = int(input())
+
+for i in range(N, 0, -1):
+    print(" "*(N-i) + "*"*(2*i-1))
```

Comment:
```suggestion
    print(" "*(N-i) + "*"*(2*i - 1))
```
파이선 코딩 컨벤션 (https://peps.python.org/pep-0008/#other-recommendations) 참고해서 작성하면 더 좋을 것 같습니다!